packages/clerk-js/src/ui/router/HashRouter.tsx

```diff
@@ -34,7 +34,14 @@ export const HashRouter = ({ preservedParams, children }: HashRouterProps): JSX.
   };
 
   const getQueryString = (): string => {
-    return fakeUrl().search;
+    // Construct a search string that combines all search params of the current url
```

Comment:
@nikosdouvlis Please add a URL example here for the future reader, as the comment is a bit hard to grasp.